instant-distance-py/examples/translations/README.md

```diff
@@ -0,0 +1,45 @@
+# Translation Example
+
+This example utilizes the pre-trained aligned word vectors made available by
+Facebook Research as part of
+[fastText](https://fasttext.cc/docs/en/aligned-vectors.html) to translate
+English words to French and Italian.
+
+## Trying it out
+
+Python 3.9 is required.
+
+First, install the requirements:
+
+```
+pip install -r requirements.txt
+```
+
+Run the following command to translate an english word:
+
+```
+python translate.py translate hello
+```
+
+```
+Loading indexes from filesystem...
+Language: fr, Translation: hello
+Language: it, Translation: hello
+Language: it, Translation: ciao
+Language: fr, Translation: bonjours
+Language: fr, Translation: bonjour
+Language: fr, Translation: bonsoir
+Language: fr, Translation: !
+Language: fr, Translation: salutations
+Language: it, Translation: buongiorno
+Language: it, Translation: hey
+```
+
+The translate command will download the vector data and build an index on the
+first run. Subsequent runs will be faster.
```

Comment:
Tell them how longs this takes on your MacBook Pro

Reply:
Decided not to include timing information as part of the script to keep it simple, and just added timing for my setup.